### PR TITLE
Bump and pin for chef-dk-0.7.0.rc.2

### PR DESCRIPTION
**Pull request by ksubrama**

### CODE REVIEW

File: config/projects/chefdk.rb [hunk 1]
```diff
@@ -66,7 +68,6 @@
 override :'ruby-windows', version: "2.1.6"
 override :'ruby-windows-devkit', version: "4.7.2-20130224-1151"
 override :'openssl-windows', version: "1.0.1m"
-#override :'ruby-windows', version: "2.0.0-p451"
```
Comment (fnichol) [inline config/projects/chefdk.rb]: The intent is to use `ruby-windows` version 2.1.6, correct?

Comment (ksubrama) [inline config/projects/chefdk.rb]: Yes.  I was cleaning out an old commented line.

Comment (fnichol) [inline config/projects/chefdk.rb]: Whoops, didn't see the `#` character

**Comment by fnichol** [comment]
:+1:

**Comment by danielsdeleo** [comment]
:+1:

File: config/projects/chefdk.rb (hunk 1 above)
Comment (ksubrama) [inline config/projects/chefdk.rb]: Worth it for the Wheaton gif.  Adding that my list of gifs to use in a hurry.

**Pull request merged**